### PR TITLE
AT_06.04_005 | <People> Edit User description

### PR DESCRIPTION
**Pull request by VeraDoa**
https://trello.com/c/mZDDnILE/1354-at0604005-people-edit-users-description

### CODE REVIEW

File: cypress/e2e/peopleEditDescription_zb.cy.js [hunk 1]
```diff
@@ -36,6 +36,20 @@ describe('People Edit Discription to User', () => {
         cy.get('button[name="Submit"]').click()
         cy.get('#description div:nth-of-type(1)').should('have.text', userDescription_zb.editOldDescription)
     })
+
+    it('AT_06.04_005 | <People> Edit User description', () => {
+        cy.get('a[href="/asynchPeople/"]').should('have.text', 'People').click()
```
Comment (Rom-T) [inline cypress/e2e/peopleEditDescription_zb.cy.js:41]: Please move `People` in the fixtures

File: cypress/e2e/peopleEditDescription_zb.cy.js [hunk 2]
```diff
@@ -36,6 +36,20 @@ describe('People Edit Discription to User', () => {
...
+        cy.get(`a[href*='/user/${USERNAME.toLowerCase()}']`).click()
```
Comment (Rom-T) [inline cypress/e2e/peopleEditDescription_zb.cy.js:42]: it's better not to bind to the username in the selector

File: cypress/e2e/peopleEditDescription_zb.cy.js (hunk 1 above)
Comment (VeraDoa) [reply cypress/e2e/peopleEditDescription_zb.cy.js]: Fixed! Thank you!

**Review by Rom-T — APPROVED** [review]

**Review by allamalitska — APPROVED** [review]

**Comment by VeraDoa** [comment]
[AT_06.04_005 | <People> Edit User's description](https://trello.com/c/mZDDnILE/1354-at0604005-people-edit-users-description)

**Comment by VeraDoa** [comment]
[AT_06.04_005 | <People> Edit User's description](https://trello.com/c/mZDDnILE/1354-at0604005-people-edit-users-description)

**Pull request merged**